### PR TITLE
refactor: VC, VP, and supporting models moved into components

### PR DESCRIPTION
**Pull request by Moopli**
- Verifiable Credentials and Presentation models are now in component/models/verifiable
- SD-JWT package is now in component/models/sdjwt
- move jsonld validator to component/models/ld
- Key fingerprint helpers and JOSE extensions moved to component/kmscrypto

All APIs in original afgo pkg/ folders can continue to be used, these now transparently refer/call the moved APIs.

### CODE REVIEW

**Review by vkubiv — APPROVED** [review]

File: component/kmscrypto/doc/jose/encrypter_decrypter_test.go [hunk 1]
```diff
@@ -28,19 +28,20 @@ import (
 	"github.com/google/tink/go/subtle"
 	"github.com/stretchr/testify/require"
 
+	jose2 "github.com/hyperledger/aries-framework-go/component/kmscrypto/doc/jose"
```
Comment (DRK3) [inline component/kmscrypto/doc/jose/encrypter_decrypter_test.go:31]: Should find a better name than `jose2`

File: component/kmscrypto/doc/util/fingerprint/parse.go [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+*/
+
+package fingerprint
+
+import (
+	"fmt"
+	"strings"
+)
+
+// TODO: should we reimplement did.Parse here, so fingerprint package no longer depends on did?
```
Comment (DRK3) [inline component/kmscrypto/doc/util/fingerprint/parse.go:13]: ?

File: component/models/jose/diddocresolver/resolver.go [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+	jsonWebKey2020            = "JsonWebKey2020"
+	x25519KeyAgreementKey2019 = "X25519KeyAgreementKey2019"
+)
+
+type vdrResolver interface {
+	Resolve(did string, opts ...vdr.DIDMethodOption) (*did.DocResolution, error)
+}
+
+// DIDDocResolver helps resolves a KMS kid from 'kid'/'skid' with values set as didDoc[].KeyAgreement[].ID. The list of
+// DIDDocs should contain both sender and recipients docs for proper resolutio during unpacking.
```
Comment (DRK3) [inline component/models/jose/diddocresolver/resolver.go:33]: typo in `resolutio`

File: component/models/jwt/jwt_support.go [hunk 4]
```diff
@@ -0,0 +1,194 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
```
Comment (DRK3) [inline component/models/jwt/jwt_support.go:2]: I suppose this copyright should be changed? @rolsonquadras

**Review by sudeshrshetty — APPROVED** [review]

**Review by DRK3 — APPROVED** [review]

**Pull request merged**